app/src/main/res/layout/inactive_header_item.xml

```diff
@@ -14,7 +14,8 @@
     android:importantForAccessibility="no"
     android:paddingStart="1dp"
     android:paddingTop="1dp"
-    android:paddingEnd="1dp">
+    android:paddingEnd="1dp"
+    android:paddingBottom="1dp">
```

Comment:
nit: should we just make it `android:padding="1dp"`?

Reply:
That works too. This was outside of my intended change but had a small effect on the visuals, so I tried to keep it minimal. 🙂 

I'll include this change if there are any other changes needed.